package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "npm run lint:src && npm run lint:build",
     "lint:src": "eslint",
     "lint:build": "npm run build firefox -- --silent && addons-linter dist",
-    "test": "npm run wdio && npm run wdio:update",
+    "test": "npm run test:unit && npm run wdio && npm run wdio:update",
+    "test:unit": "node --test",
```

Comment:
If we can't run like `node something /*.test.js`, add `tests/unit/index.js` and run this file.